task_12/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "task_12",
+	"version": "1.0.0",
+	"description": "SPA as part of kottans/frontend course ",
+	"main": "index.js",
+	"scripts": {
+		"dev": "webpack",
+		"production": "NODE_ENV=production webpack",
+		"watch": "npm run dev -- --watch"
+	},
+	"babel": {
+		"presets": [
+			"env"
+		]
+	},
+	"author": "@val-fom",
```

Comment:
Field `author` is well specified by `package.json` docs. Read about [people fields](https://docs.npmjs.com/files/package.json#people-fields-author-contributors).